extensions/graphite-emitter/README.md

```diff
@@ -0,0 +1,50 @@
+## introduction
+
+This extension emits some selected druid metrics to a graphite carbon server.
+Events are sent after been [pickled](http://graphite.readthedocs.org/en/latest/feeding-carbon.html#the-pickle-protocol) , the size of the batch is configurable.
+Since Graphite doesn't support alerting, alerts will be redirect to warning logs by default. 
+
+ Metrics are organized using the following schema:
+`<namespacePrefix>.[<druid service name>].[<druid hostname>].<druid metrics dimensions>.<druid metrics name>`
+Properly naming your metrics is critical to avoid conflicts, confusing data and potentially wrong interpretation later on.
+
+Example `druid.historical.hist-host1.yahoo.com:8080.MyDataSourceName.GroupBy.query/time`:
+
+ * `druid` -> namespace prefix 
+ * `historical` -> service name 
+ * `hist-host1.yahoo.com:8080` -> druid hostname
+ * `MyDataSourceName` -> dimension value 
+ * `GroupBy` -> dimension value
+ * `query/time` -> metric name
+
+User has control of `<namespace>.[<druid service name>].[<druid hostname>].`
+
+You can omit the hostname by setting `druid.emitter.graphite.ignoreHostname=true`
+`druid.SERVICE_NAME.dataSourceName.queryType.query.time`
+You can omit service name by setting `druid.emitter.graphite.ignoreServiceName=true`
+`druid.HOSTNAME.dataSourceName.queryType.query.time`
```

Comment:
It would be better if these lines weren't all run together.  Maybe you could say
You can emit the hostname by setting druid.emitter.graphite.ignoreHostname=true, resulting in this:
  druid.SERVICE_NAME.dataSourceName.queryType.query.time 
You can omit the service name by setting druid.emitter.graphite.ignoreServiceName=true, result in this:
  druid.HOSTNAME.dataSourceName.queryType.query.time

(Note you are missing "the" in "emit service name".)